src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+  public static void main(String[] args) {
+    Machine[] machines = new Machine[]{
+        new Bulldozer(),
+        new Excavator(),
+        new Truck()
+    };
+    for (Machine machine : machines) {
```

Comment:
Imagine you are building some building and you need to all machines work at the same time.
But in your code, machine working and stops. After that other machine works and stop.
So, it's need to rewrite this part to make all machine work. After that all machine stop working.